leshan-core/src/test/java/org/eclipse/leshan/core/node/codec/LwM2mNodeDecoderTest.java

```diff
@@ -1277,24 +1277,12 @@ public void senml_multiple_timestamped_nodes() throws CodecException {
                 model);
 
         // then
-        Map<Long, Map<LwM2mPath, LwM2mNode>> expectedResult = new HashMap<>();
-        Map<LwM2mPath, LwM2mNode> first = new HashMap<>();
-        first.put(new LwM2mPath("/3/0/7/0"), LwM2mResourceInstance.newIntegerInstance(0, 3800));
-        expectedResult.put(268600000L, first);
+        TimestampedLwM2mNodes.Builder expectedResult = new TimestampedLwM2mNodes.Builder();
+        expectedResult.put(268600000L, new LwM2mPath("/3/0/7/0"), LwM2mResourceInstance.newIntegerInstance(0, 3800));
+        expectedResult.put(268600001L, new LwM2mPath("/4/0/0"), LwM2mSingleResource.newIntegerResource(0, 1));
+        expectedResult.put(268600002L, new LwM2mPath("/4/0/1"), LwM2mSingleResource.newIntegerResource(1, 2));
+        expectedResult.put(268600003L, new LwM2mPath("/4/0/2"), LwM2mSingleResource.newIntegerResource(2, 3));
```

Comment:
You can use fluency of builder eg:
```java
   TimestampedLwM2mNodes expectedResult = new TimestampedLwM2mNodes.Builder()
      .put(268600000L, new LwM2mPath("/3/0/7/0"), LwM2mResourceInstance.newIntegerInstance(0, 3800))
      .put(268600001L, new LwM2mPath("/4/0/0"), LwM2mSingleResource.newIntegerResource(0, 1))
      .put(268600002L, new LwM2mPath("/4/0/1"), LwM2mSingleResource.newIntegerResource(1, 2))
      .put(268600003L, new LwM2mPath("/4/0/2"), LwM2mSingleResource.newIntegerResource(2, 3))
      .build()
```

Reply:
By the past I have some formatting issue with this kind of notation, so I get out the habit to use it.
But I just test it and it sounds good this time, so I will change it.